Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift

```diff
@@ -50,6 +47,8 @@ struct SentryUserFeedbackWidget {
                     constraints.append(button.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: config.widgetConfig.layoutUIOffset.horizontal))
                 }
                 NSLayoutConstraint.activate(constraints)
+                
+                NotificationCenter.default.addObserver(self, selector: #selector(userCapturedScreenshot), name: UIApplication.userDidTakeScreenshotNotification, object: nil)
```

Comment:
`h`: If we need to subscribe to this, we also must remove the observer at some point.